### DISCUSSION TITLE
Probat 12/2 (7" Screen) PID basic starting point?

### DISCUSSION THREAD

**Discussion by RNCCoffee** [artisan-roaster-scope/artisan#1276] [Q&A]
Anyone know where I can get some basic instructions to get a starting point for PID (Artisan) for our setup? We're looking to follow the background, but enable PID to do so. 

Not looking for the fine tuning (yet) just a basic starting point. I've look through the docs, but hoping for a starting point that doesn't have me testing 11kg batches with random settings...

I tried to 'idle' the machine at our charge temp, it kept rocketing way past the temp then turning off the burner (repeat, repeat repeat).  It never did settle. I expect that I had some settings incorrect but am having trouble determining what those were). 

Caveat: I am pretty familiar with the basic Artisan Scope interface but admit that I'm barely understanding PID beyond what it's 'supposed' to do. 

Thanks in advance for any resources that I can be pointed to. 

Cheers!

Jamie

**Comment by CarefreeBuzzBuzz** [comment]
Honestly, if you are roasting on that, just hire Marko to get you up to speed immediately. no sense in wasting your time or coffee when you could have it running like a well tuned PorscheConsultingshop.artisan.plus___________________________Michael Herbert, CarefreeBuzzBuzzOn Oct 3, 2023, at 8:44 AM, RNCCoffee ***@***.***> wrote:﻿
Anyone know where I can get some basic instructions to get a starting point for PID (Artisan) for our setup? We're looking to follow the background, but enable PID to do so.
Not looking for the fine tuning (yet) just a basic starting point. I've look through the docs, but hoping for a starting point that doesn't have me testing 11kg batches with random settings...
I tried to 'idle' the machine at our charge temp, it kept rocketing way past the temp then turning off the burner (repeat, repeat repeat).  It never did settle. I expect that I had some settings incorrect but am having trouble determining what those were).
Caveat: I am pretty familiar with the basic Artisan Scope interface but admit that I'm barely understanding PID beyond what it's 'supposed' to do.
Thanks in advance for any resources that I can be pointed to.
Cheers!
Jamie

**Comment by RNCCoffee** [comment]
Who's Marko?

**Comment by CarefreeBuzzBuzz** [comment]
He and Dave are the two primary software engineers. Nobody can help you
better than he can.

**Comment by RNCCoffee** [comment]
Got it. Thnx!

**Comment by RNCCoffee** [comment]
Sadly, it's not worth the $400 for me to hire someone. Artisan works as it is for us on the basic mode, I was just looking at PID to augment what we already do. I'll keep searching.

Cheers!

**Reply by MAKOMO** [reply]
Sad. Increase d and decrease p, set i to zero or close to zero.

**Reply by RNCCoffee** [reply]
No offence intended. Artisan works exceptionally well for us already. Introducing PID into the mix is more of a 'nice-to-have' as opposed to a 'need'.
I'm not opposed to paying for help when needed, but this isn't a priority at the moment. If it becomes a bigger part of our operation, I'll certainly reach out. 
Thank you for the advice. I'll give it a try later this week.
